packages/common/src/tokens/utils.ts

```diff
@@ -24,3 +56,86 @@ export function sortByAddress<T extends Token | TokenObject | TokenAmount | Toke
     return address.toLowerCase();
   });
 }
+
+const basicTokenMap: Record<number, Record<string, Token>> = {
```

Comment:
```suggestion
const unifiedTokenMap: Record<number, Record<string, Token>> = {
```

also modify the changeset & PR name

Reply:
updated